plugins/obs-webrtc/webrtc/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "webrtc-rs-lib"
+version = "0.1.0"
+edition = "2021"
+build = "build.rs"
+
+[dependencies]
+webrtc = "0.6.0"
+webrtc-mdns = "0.5.2"
+tokio = { version = "1.16.1", features = ["full"] }
+anyhow = "1.0.45"
+bytes = "1"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+serde_derive = "1.0"
```

Comment:
this `serde_derive` dependency appears to be unnecessary

Reply:
Good catch

Reply:
Fixed in 10bca592d180032d8c836a127a60429f875be89b